exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/ServiceRuntimeConfigurationIntegrationTest.java

```diff
@@ -81,9 +82,10 @@ void runtimeConfigurationTest(@TempDir Path tmpArtifactDir) throws Exception {
       // Initialize and register a service instance
       String name = "s1";
       ServiceInstanceSpec instanceSpec = ServiceInstanceSpec.newInstance(name, 1, ARTIFACT_ID);
+      InstanceStatus instanceStatus = InstanceStatus.newBuilder().setSimple(Simple.NONE).build();
       Fork fork = database.createFork(cleaner);
       runtime.initiateAddingService(fork, instanceSpec, new byte[0]);
-      runtime.updateInstanceStatus(instanceSpec, InstanceState.Status.NONE);
+      runtime.updateInstanceStatus(instanceSpec, instanceStatus);
```

Comment:
Why NONE? That must be an artifact between the Exonum native update and stopping support — it must be active, I believe.

Reply:
yeap, I'll change during merge with stopping services